specification/translation/Azure.AI.DocumentTranslation/models.tsp

```diff
@@ -409,12 +409,6 @@ model DocumentsStatus {
   nextLink?: string;
 }
 
-@doc("List of supported file formats")
-model SupportedFileFormats {
```

Comment:
Why are we deleting this?

Reply:
Because we updated the getSupportedFormats api to return FileFormat[] instead of the SupportedFileFormats model

Reply:
Reverted this change